tests/resources/serializers/test_marcxml_serializer.py

```diff
@@ -74,6 +74,7 @@ def test_marcxml_serializer_minimal_record(running_app, minimal_record, parent):
     expected_data = f"""
         <?xmlversion='1.0'encoding='utf-8'?>
         <record xmlns="http://www.loc.gov/MARC21/slim">
+            <leader>00000nam##2200000uu#4500</leader>
```

Comment:
This also loks correct, see e.g. the output from https://zenodo-dev.cern.ch/oai2d?verb=GetRecord&metadataPrefix=marc21&identifier=oai:zenodo.org:59917